Detectors/Vertexing/src/SVertexer.cxx

```diff
@@ -570,6 +575,7 @@ bool SVertexer::checkV0(const TrackCand& seedP, const TrackCand& seedN, int iP,
   auto& fitterV0 = mFitterV0[ithread];
   // Fast rough cuts on pairs before feeding to DCAFitter, tracks are not in the same Frame or at same X
   bool isTPConly = (seedP.gid.getSource() == GIndex::TPC || seedN.gid.getSource() == GIndex::TPC);
+  bool isITSOBonly = (seedP.bITSOB || seedN.bITSOB);
```

Comment:
Below (L670) you interpret `isITSOBonly` as not only (1) ITS hits being only on the OB but also as (2) not having TPC. 
But to set the bITSOB flag it is enough to fulfill (1) only. Either one should check here additionally for `seedP.gid.getSource() == GIndex::ITS ...` or exclude tracks with TPC in the assignment of the `bITSOB` in the `buildT2V`.